pkg/infra/usagestats/service/api.go

```diff
@@ -0,0 +1,27 @@
+package service
+
+import (
+	"net/http"
+
+	"github.com/grafana/grafana/pkg/api/response"
+	"github.com/grafana/grafana/pkg/api/routing"
+	"github.com/grafana/grafana/pkg/middleware"
+	"github.com/grafana/grafana/pkg/models"
+)
+
+const rootUrl = "/api/admin"
+
+func (uss *UsageStats) registerAPIEndpoints() {
+	uss.RouteRegister.Group(rootUrl, func(subrouter routing.RouteRegister) {
```

Comment:
I know I am a late on this, but I think we need to have FGAC permissions on this endpoint too. As we are already restricting the Admin Stats page and relevant endpoints.

Reply:
As we talked in Slack, I was confused with server stats and usage stats. As this is about usage stats, we don't have yet fine-grained permissions on it, so should be fine to leave it like this for the time being.